tasks/connectors/qualys_was/qualys_was.rb

```diff
@@ -230,7 +230,7 @@ def domain_detail(find_from)
       end
 
       def name(find_from)
```

Comment:
Good encapsulation, @effany, moving the code in the method. I recommend avoiding having `-` when you have nil values in the parms to refactor the code to something like this and maybe adding some defensive code for nil param find_from or make a code fragment that generates a name if you know that rule for its creation. 

```
def name(find_from)
  return if find_from.nil? # or other created on the flight name 
  structured_name = [find_from['name'], find_from['qid'], find_from['id']].compact.join(' - ')
  structured_name unless structured_name.empty?
end
```